generators/app/templates/service/package.json

```diff
@@ -37,6 +37,7 @@
   },
   "homepage": "https://github.com/<%= orgName %>/<%= projectName %>#readme",
   "dependencies": {
+    "@buttercup/generator": "^1.2.0",
```

Comment:
This dependency should be conditional. It should be installed only when `commerce` is installed. Similar to `@cxcloud/ct-types`. Maybe find a solution for this in `index.js` of the generator where we have that `installing` function